upgrades/upgrade-1.6-2.0.rst

```diff
@@ -0,0 +1,291 @@
+Upgrading from Sulu 1.6 to 2.0
+==============================
+
+We have a very extensive `UPGRADE.md file`_ in our repository, which contains almost every breaking change we made
+during the development of Sulu 2.0. However, this is a huge list and most projects won't be affected by the majority of
+these breaking changes. Therefore we are going through the most important changes in this guide, and by following these
+steps you can probably upgrade most of the projects, if you have not overridden some internal mechanisms.
```

Comment:
Please be aware that this guide does not contain any steps that are needed if your project overrides sulu internal mechanisms.

Reply:
Same as above, avoiding at least one `that` :see_no_evil: